src/order/pfilter.lean

```diff
@@ -32,6 +34,8 @@ pfilter, filter, ideal, dual
 
 -/
 
+namespace order
```

Comment:
Yes, I think it makes sense for order ideals and order filters to live in the same namespace (rather than `pfilter` being in the root and `ideal` being in `order`).

However, this does mean that every instance of `order.ideal` in this file should be changed to just `ideal` (except for the import statement at the very top). I'll let you do the search & replace rather than adding a bunch of suggested changes in the review.